pkgs/applications/misc/furtherance/default.nix

```diff
@@ -3,19 +3,19 @@
 
 stdenv.mkDerivation rec {
   pname = "furtherance";
-  version = "1.6.0";
+  version = "1.7.0";
 
   src = fetchFromGitHub {
     owner = "lakoliu";
     repo = "Furtherance";
-    rev = "v${version}";
-    sha256 = "xshZpwL5AQvYSPoyt9Qutaym5IGBQHWwz4ev3xnVcSk=";
+    rev = "dff62a9e31ff85305eb2d70b036ee25c90e4feeb";
```

Comment:
Just a question: why not keep using the tag as rev? I think the upstream project is correctly tagged.

Reply:
Well the reason is because v1.7.0 doesn't have the cargo dependencies updated, so it wont build, it was the dependency after that which introduced the fix that makes it build without issues, when 1.7.x comes out I'll put it back to version instead of rev.